### DISCUSSION TITLE
Hide connection window on Windows

### DISCUSSION THREAD

**Discussion by flashgordonAR** [rustdesk/rustdesk#6552] [Q&A] [answered]
Hi, There used to be an option in the Security tab, when you set up a fixed password to connect to a device, where the Rustdesk window could be hidden.  That option is not there anymore.  Will it be added in the future or was there any problem to hide that window?

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/6008
